docs/reagent-integration.md

```diff
@@ -0,0 +1,112 @@
+# Reagent Integration
```

Comment:
I would title this document "Integrating with other libraries" and have a subheading for Reagent and re-frame.